Source/CarthageKit/Errors.swift

```diff
@@ -188,6 +191,9 @@ extension CarthageError: Printable {
 		case let .InvalidArchitectures(description):
 			return "Invalid architecture: \(description)"
 
+		case let .InvalidUUIDs(description):
+			return "Invalid UUIDs: \(description)"
```

Comment:
This error isn't at all obvious. It'd be nice if it could mention a specific project and mention something about binary releases and which file has problems.